examples/health-check/src/Definition.hs

```diff
@@ -34,27 +35,31 @@ newtype HealthCheckMsg
   = HealthCheckMsg { nameService :: T.Text }
   deriving ( Eq, Show, Ord, Generic
            , ToSchema   HealthCheckSchema "HealthCheck"
-           , FromSchema HealthCheckSchema "HealthCheck" )
+           , FromSchema HealthCheckSchema "HealthCheck"
+           , Swagger.ToSchema )
```

Comment:
hahahaha amazing, just by adding `deriving Swagger.ToSchema` we get all this magic! 🚀